client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/CheckAnticipatedStatusChangeSupportedParams.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.backend.issue;
+
+public class CheckAnticipatedStatusChangeSupportedParams {
+  private final String connectionId;
+
+  public CheckAnticipatedStatusChangeSupportedParams(String connectionId) {
+    this.connectionId = connectionId;
```

Comment:
Later this feature will also be supported in standalone mode, so maybe passing the `connectionId` is not ideal, and we should pass the `configScopeId`, that the backend is then able to resolve to a connection

Reply:
Done, so we don't have to change the API in the future.